save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt

```diff
@@ -43,7 +43,7 @@ private fun ChildrenBuilder.displayTopProjects(sortedResults: List<ContestResult
                 "${index + 1}. ",
                 contestResult.projectName,
                 contestResult.organizationName,
-                contestResult.score?.toString() ?: "-",
+                (contestResult.score?.toString().asDynamic().toFixed(2) as String? ?: "-"),
```

Comment:
Why `asDynamic()`?

Reply:
Kotlin native have no methods for decimal precisions, but kotlin JS does: https://stackoverflow.com/questions/42791492/kotlin-js-define-number-of-decimals

Reply:
ok, weird that Kotlin does not have it

Reply:
Only kotlin jvm does, other seems to be absent and issue is still opened for more than 5 years https://youtrack.jetbrains.com/issue/KT-21644